### PR TITLE
PLAT-615 Show Role Names, Not UUIDs

### PR DESCRIPTION
**Pull request by thees-k**
* FIXME was added

* Test Plan: Just view the change﻿
Screenshot_2022-01-05_11-55-36

### CODE REVIEW

File: platform-core-module/src/main/java/com/softicar/platform/core/module/uuid/AGUuid.java [hunk 1]
```diff
@@ -29,7 +32,17 @@ public IDisplayString toDisplayWithoutId() {
...
+	}
+
+	// FIXME Should be replaced when roles can be created at runtime
+	private static IDisplayString getStaticRoleTitle(UUID uuid) {
+
+		return CurrentEmfRoleRegistry//
+			.get()
+			.getStaticRole(uuid)
+			.map(IEmfStaticRole::getTitle)
+			.orElse(DisplayString.EMPTY);
```
Comment (daniel-klose) [inline platform-core-module/src/main/java/com/softicar/platform/core/module/uuid/AGUuid.java:45]: should be
`.orElse(IDisplayString.create(getUuidString()));`

File: platform-core-module/src/main/java/com/softicar/platform/core/module/uuid/AGUuid.java [hunk 2]
```diff
@@ -29,7 +32,17 @@ public IDisplayString toDisplayWithoutId() {
 		return EmfSourceCodeReferencePoints//
 			.getReferencePoint(getUuid())
 			.map(IEmfSourceCodeReferencePoint::toDisplay)
-			.orElse(IDisplayString.create(getUuidString()));
+			.orElse(getStaticRoleTitle(getUuid()));
+	}
+
+	// FIXME Should be replaced when roles can be created at runtime
+	private static IDisplayString getStaticRoleTitle(UUID uuid) {
```
Comment (daniel-klose) [inline platform-core-module/src/main/java/com/softicar/platform/core/module/uuid/AGUuid.java:39]: method doesnt need to be static
Also, should be moved to bottom below `private static` methods

**Review by daniel-klose — CHANGES_REQUESTED** [review]

File: platform-core-module/src/main/java/com/softicar/platform/core/module/uuid/AGUuid.java (hunk 1 above)
Comment (thees-k) [reply platform-core-module/src/main/java/com/softicar/platform/core/module/uuid/AGUuid.java:45]: ok

File: platform-core-module/src/main/java/com/softicar/platform/core/module/uuid/AGUuid.java (hunk 2 above)
Comment (thees-k) [reply platform-core-module/src/main/java/com/softicar/platform/core/module/uuid/AGUuid.java:39]: ok and ok

**Review by daniel-klose — APPROVED** [review]

**Pull request merged**